custom_components/tuya_local/devices/pc311ty_energy_meter.yaml

```diff
@@ -0,0 +1,222 @@
+name: PC311-TY power clamp
+products:
+  - id: vfwxetkavqrvg02u
+primary_entity:
+  entity: sensor
+  category: diagnostic
+  class: power
+  name: Total active power
```

Comment:
For an energy meter, would it make more sense for the Total Energy reading to be the primary sensor?

Reply:
I changed the primary sensor